examples/testimonials/readme.md

```diff
@@ -0,0 +1,15 @@
+# Forms example - Testimonial
+This is an example of using a form submission to automatically create a piece type in your Apostrophe 3 project.
+## Installation
+* Copy the `modules/@apostrophe/form` folder into your project
```

Comment:
There is a [web site](https://download-directory.github.io/), that allows you to download individual folders from GitHub as zip files. Maybe we could add such a link for each folder to be downloaded. We should just add a small description of what will happen when you click on the link so that people aren't surprised.